DfciPkg/SettingsManager/SettingsManagerProvider.c

```diff
@@ -376,19 +372,23 @@ ProviderValueAsAscii (
         break;
       }
 
-      Value = AllocateZeroPool (ENABLED_STRING_SIZE);
+      if (v == ENABLE_INCONSISTENT) {

```

Comment:
Note that I submitted issue [#45](https://github.com/microsoft/mu_feature_dfci/issues/45) to track changing the type of `v` to be non-`BOOLEAN`.

Looks like @apop5 plans to fix that in a separate change.